src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -633,6 +637,7 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
       throw new Error('performWork was called recursively.');
     }
     isPerformingWork = true;
+    isPerformingDeferredWork = priorityLevel >= HighPriority;
```

Comment:
HighPriority is scheduled with a deferred callback, not an animation callback. This was the reason for the name change a while back.